CITATION.cff

```diff
@@ -0,0 +1,23 @@
+#
+# This source file is part of the ResearchKitOnFHIR open source project
+#
+# SPDX-FileCopyrightText: 2022 Stanford Biodesign for Digital Health and the project authors (see CONTRIBUTORS.md)
+#
+# SPDX-License-Identifier: MIT
+#
+
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
```

Comment:
It is conventional in academic citation to order by contribution, as seen here: https://github.com/StanfordBDHG/ResearchKitOnFHIR/graphs/contributors

Reply:
Thank you, sounds great!
I am unsure if the GitHub contributors page is generally the best metric as it sorts by commits. Commits can be very large or very small and are not the best metric to calculate contribution.
Nevertheless, I am fine with you being listed first; it is a fair recognition for all your commits and time spent on the project! 🚀

Reply:
Thanks Paul, I'm not too stuck on being listed first - the intent of my comment here was to make sure we adopt processes that ensure every team member is fairly recognized for their contributions throughout our projects. I realize the Github contributors page is not the best metric there is and I'm open to other ideas you have!